src/osemgrep/cli_interactive/Framed_zipper.mli

```diff
@@ -43,6 +43,7 @@ type 'a t
 val move_left : 'a t -> 'a t
 val move_right : 'a t -> 'a t
 val of_list : int -> 'a list -> 'a t
+val append : 'a -> 'a t -> 'a t
```

Comment:
maybe add a few comments in the .mli about those different functions (Does not have to be very long).
For example for append maybe explain what's the semantic of it if the frame is at the end? does the frame move also if you add a new element?